src/ImageCropBundle/Bridge/Chameleon/MediaManager/Mapper/ImageCropEditorUrlMapper.php

```diff
@@ -70,4 +79,18 @@ private function getCropEditorUrl($mediaItemId)
 
         return URL_CMS_CONTROLLER.$this->urlUtil->getArrayAsUrl($parameters, '?', '&');
     }
+
+    /**
+     * @return string
+     */
+    private function getDeleteCropUrl()
```

Comment:
```suggestion
    private function getDeleteCropUrl(): string
```

(docblock can then be removed)

Reply:
OK